src/loci/formats/in/ZarrReader.java

```diff
@@ -1226,4 +1242,103 @@ private void reloadOptionsFile(String id) {
       }
     }
   }
+  
+  public static byte[] openPlane(String path, byte[] buf) throws FormatException, IOException {
```

Comment:
Is this new static API in use anywhere? Or was it primarily used for profiling?

Reply:
It is not currently in use anywhere, it was used in the testing and profiling but never implemented anywhere. I will remove it from this PR.